Dockerfile

```
@@ -5,7 +5,12 @@ LABEL source=https://github.com/Arxcis/adventofcode2020
 ENV TZ=Europe/Oslo
 RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
 
-# 2. apt install all the things
+# Configure repository for zig
+# See: https://github.com/dryzig/zig-debian/blob/master/README.md
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 379CE192D401AB61
```

I had to install gpg, to be able to add a keyserver:
```suggestion
RUN apt-get update && apt-get install -yqq gpg && \
        apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 379CE192D401AB61

```

---

Also the key hardcoded here is outdated, and not accepted by gpg.